subsys/bluetooth/shell/audio.c

```diff
@@ -1409,6 +1409,33 @@ static int cmd_term_broadcast_sink(const struct shell *sh, size_t argc,
 }
 #endif /* CONFIG_BT_AUDIO_BROADCAST_SINK */
 
+static int cmd_set_loc(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err;
+	enum bt_audio_dir dir;
+	enum bt_audio_location loc;
+
+	ctx_shell = sh;
+
+	if (!strcmp(argv[1], "sink")) {
+		dir = BT_AUDIO_DIR_SINK;
+	} else if (!strcmp(argv[1], "source")) {
+		dir = BT_AUDIO_DIR_SOURCE;
+	} else {
+		shell_error(sh, "Unsupported dir: %s", argv[1]);
+		return -ENOEXEC;
+	}
+
+	loc = shell_strtoul(argv[2], 16, &err);
```

Comment:
Remember to initialize `err` to `0` first :)

Reply:
Done